tests/integration/cmd_pref_config_test.go

```diff
@@ -65,10 +65,7 @@ var _ = Describe("odo preference and config command tests", func() {
 		})
 		It("should get the default global config keys", func() {
 			configOutput := helper.CmdShouldPass("odo", "preference", "view")
-			Expect(configOutput).To(ContainSubstring("UpdateNotification"))
-			Expect(configOutput).To(ContainSubstring("NamePrefix"))
-			Expect(configOutput).To(ContainSubstring("Timeout"))
-			Expect(configOutput).To(ContainSubstring("PushTarget"))
+			helper.MatchAllInOutput(configOutput, []string{"UpdateNotification", "NamePrefix", "Timeout", "PushTarget"})
```

Comment:
Please format this line as its more then 120 characters

Reply:
> Please format this line as its more then 120 characters

Its hard time for me to understand what you are pointing to. Can you please give a example or a reference would really be helpful

Reply:
Make this command into multiple lines as a single line of code shouldn’t be more then 120 characters

Reply:
I am talking about the line mentioned in the comment below this git comment